client/components/Nav.jsx

```diff
@@ -20,23 +20,20 @@ function Nav() {
   }
 
   return (
-    <>
-      <NavGroup>
-        <NavLink to="/">Home</NavLink>
-        <IfAuthenticated>
-          <NavLink to="/" onClick={handleLogOff}>
-            Log off
-          </NavLink>
-          <p>{user.username}</p>
-        </IfAuthenticated>
-        <IfNotAuthenticated>
-          <NavLink to="/" onClick={handleSignIn}>
-            Sign In
-          </NavLink>
-        </IfNotAuthenticated>
-      </NavGroup>
-      <h1>Fruit FTW!</h1>
-    </>
+    <div>
+      <NavLink to="/">Home</NavLink>
+      <IfAuthenticated>
+        <NavLink to="/" onClick={handleLogOff}>
+          Log off
+        </NavLink>
+        <p>{user.username}</p>
```

Comment:
If your using material then use the `Paragraph` component rather than a pure p tag. Your theme styles wont get effected here